.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+# The name of the workflow
+    name: build
+    
+    # What conditions trigger the workflow
+    #  In this case, all pushes and pull requests
+    on: [push, pull_request]
+    
+    # The jobs that will be run, usually in parallel
+    jobs:
+      # A job to generate and publish code coverage
+      coverage:
+    		# A more descriptive name of the job
```

Comment:
Red text indicates tabs instead of spaces... @patrick-replogle, please remove tabs or replace tabs with spaces.

Reply:
I updated the spacing and it took out the red text and it seems to like it now. I didn't catch that since I'm so used to prettier making things...well, prettier, but it doesn't seem to watch this file.